### PR TITLE
BOCM-288 test created for GET /inspector/more-appeals

### PR DESCRIPTION
**Pull request by MariaValeroGonzalezKC**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [√ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by mayya-bond-kc — COMMENTED** [review]
Overall looks good. Some questions I have are around the response, it doesn't quite match what the requirements are in the ticket

File: apps/api/src/server/app/inspector/inspector.controller.js [hunk 1]
```diff
@@ -63,6 +66,14 @@ const getAppeals = async function(request, response) {
 	return response.send(appealsForResponse);
 };
 
+const getMoreAppeals = async function(request, response) {
+	const moreAppeals = await appealRepository.getByStatuses(
+		[inspectorStatesStrings.available_for_inspector_pickup]
+	);
+	console.log('HERE', moreAppeals);
```
[the remark this answers is not in the document]
Comment (MariaValeroGonzalezKC) [reply apps/api/src/server/app/inspector/inspector.controller.js:73]: done

File: apps/api/src/server/app/inspector/inspector.get-more-appeals.test.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+			reference: 'APP/Q9999/D/21/5463281',
+			status: 'available_for_inspector_pickup',
+			address: {
+				addressLine1: '56 Vincent Square',
+				county: 'London',
+				postCode: 'SW1P 2NE'
+			},
+			appealAge: 22,
+			siteVisit: {},
+			siteVisitType: 'accompanied',
```
[the remark this answers is not in the document]
Comment (MariaValeroGonzalezKC) [reply apps/api/src/server/app/inspector/inspector.get-more-appeals.test.js:58]: corrected

**Comment by MariaValeroGonzalezKC** [comment]
too many issues with conflict files

**Pull request closed without merging**